apps/site/components/Filter/FilterFixedUsage.tsx

```diff
@@ -0,0 +1,100 @@
+import { setFacet, useSearch } from '@faststore/sdk'
+
+import {
+  FacetBoolean as UIFacetBoolean,
+  FacetBooleanItem as UIFacetBooleanItem,
+  FacetRange as UIFacetRange,
+  Facets as UIFacets,
+  Filter as UIFilter,
+} from '@faststore/ui'
+import { useFormattedPrice } from '../utilities/usePriceFormatter'
+import { FilterFacet } from './FilterUsage'
+import { useFilter } from './useFilter'
+
+interface Props {
+  /**
+   * The array that represents the details of every facet.
+   */
+  facets: FilterFacet[]
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
```

Comment:
Since this is just an example, we don't need the `testId` prop here right?

Reply:
You are right. But we use it to forward the props for the component, so I think it's important to keep it.